### ISSUE TITLE
The second time the game virtual button layout is messed up

### ISSUE THREAD

**Issue by Cr4d** [nikita36078/J2ME-Loader#498]
**Emulator version:**1.4.4

**Game version:**almost all games

**Game resolution:**240*320Horizontal screen
``(For example, 240x320 or 640x360)``

**Device:**meizu s6
``(For example, Samsung Galaxy S7)``

**Android version:**

**Description of the issue:**
``(What's the problem? - Screenshots showing the issue if applicable)``
Move the button on the right to the left,then exit the game and restart,the button position on the left is messed up.

**Comment by Cr4d** [comment]
android version7.0

**Comment by Cr4d** [comment]
I tested it again,the directional arrow will move to the wrong position on the left after zooming in.

**Comment by Cr4d** [comment]
And  other buttons are normal.

**Comment by nikita36078** [comment]
Can you record a video with this bug? I can't reproduce it.

**Comment by Cr4d** [comment]
------------------ 原始邮件 ------------------
发件人: "Nikita Shakarun"<notifications@github.com>
发送时间: 2019年5月2日(星期四) 晚上8:47
收件人: "nikita36078/J2ME-Loader"<J2ME-Loader@noreply.github.com>;
抄送: "Cr4d"<3466483474@qq.com>;"State change"<state_change@noreply.github.com>;
主题: Re: [nikita36078/J2ME-Loader] The second time the game virtual buttonlayout is messed up (#498)



Can you record a video with this bug? I can't reproduce it.
 
&mdash;

**Comment by Cr4d** [comment]
Can you see the video I sent?

**Comment by nikita36078** [comment]
No, please send it at my email (j2me.loader@mail.ru), Github doesn't support video attachments.

**Comment by nikita36078** [comment]
Ah, you sent it at my gmail, I got it.
